python/paddle/fluid/optimizer.py

```diff
@@ -324,8 +324,6 @@ def minimize(self,
             params_grads = append_backward(loss, parameter_list, no_grad_set,
                                            [error_clip_callback])
 
-            params_grads = sorted(params_grads, key=lambda x: x[0].name)
```

Comment:
Why remove sorted in Python?

Reply:
This sort will disturb the reduction order.